integration/testdata/skaffold-in-cluster/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM gcr.io/gcp-runtimes/ubuntu_16_0_4
+
+COPY ./skaffold /usr/bin/skaffold
```

Comment:
Maybe move the test-setup `COPY` commands after the block of commands for `kubectl` to improve layer caching?